src/pages/common-language-guide/wasi-http.mdx

```diff
@@ -106,11 +129,12 @@ Next the api needs to be uploaded and deployed to Golem. This can be done using
    }
    ```
 
-2. Now we can deploy the api to our host (localhost in this example):
-
-   ```shell
-   $ golem api deployment deploy --definition example-api/0.1.0 --host localhost:9093
+2. Deploy the api to our target host (localhost in this example):
 
+   ```shell copy
+   golem api deployment deploy --definition example-api/0.1.0 --host localhost:9093
```

Comment:
Looks like the default port is `9006` if you run `golem server run` so let's use that in the examples